### PR TITLE
Update audit-logs.md

### PR DESCRIPTION
**Pull request by BeateRixen**
Added on sentence/link on audit logs for Analytics Builder, (even if the section does not contain detailed information yet).

### CODE REVIEW

**Review by rssag — APPROVED** [review]

**Review by kailol-sag — APPROVED** [review]
You're not linking directly to the "Viewing the audit logs" subsection, but I assume that this is intentional. So that's ok for me.

**Comment by BeateRixen** [comment]
> You're not linking directly to the "Viewing the audit logs" subsection, but I assume that this is intentional. So that's ok for me.

Yes, that is intentional as the text on audit logs does not help that much. We could also rephrase the text to "For information on Analytics Builder monitoring including audit logs ..." ? @kailol-sag

**Comment by kailol-sag** [comment]
> Yes, that is intentional as the text on audit logs does not help that much. We could also rephrase the text to "For information on Analytics Builder monitoring including audit logs ..." ? @kailol-sag

Sounds like a good idea :-) Yep, let's do this.

File: content/standard-tenant/audit-logs.md [hunk 1]
```diff
@@ -297,3 +297,7 @@ Note that entries of this type are not created when using Basic authentication.
 
 </tbody>
 </table>
+
+{{< c8y-admon-info >}}
+For information on Analytics Builder audit logs, see [Monitoring and configuration](/streaming-analytics/analytics-builder/#monitoring-and-configuration).
```
Comment (BeateRixen) [inline content/standard-tenant/audit-logs.md:302]:
```suggestion
For information on Analytics Builder monitoring including audit logs, see [Monitoring and configuration](/streaming-analytics/analytics-builder/#monitoring-and-configuration).
```

**Pull request merged**